src/components/BooksForm.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
- [ ] `BookForm` should be placed in container folder, since it also need to connect to redux store in the next milestones (adding books to the store).